lib/ClampedText/ClampedText.js

```diff
@@ -0,0 +1,166 @@
+require('moonstone');
+
+var
+	dom = require('enyo/dom'),
+	kind = require('enyo/kind'),
+	Control = require('enyo/Control');
+
+/**
+* Limits the display of {@link enyo.Control#content} to {@link moon.ClampedText#maxLines}
+*
+* ```
+* {kind: ClampedText, content: 'some rather long content', maxLines: 3}
+* ```
+*
+* @class moon.ClampedText
+* @ui
+* @module moonstone/ClampedText
+* @public
+*/
+module.exports = kind(
+	/** @lends moon.ClampedText.prototype */ {
+
+	/**
+	* @private
+	*/
+	name: 'moon.ClampedText',
+	
+	/**
+	* @private
+	*/
+	kind: Control,
+	
+	/**
+	* @private
+	*/
+	classes: 'moon-clamped-text',
+
+	/**
+	* @lends  moon.ClampedText.prototype
+	* @private
+	*/
+	published: {
```

Comment:
Yeah, we're moving away from them. There was a bit of wavering on this on the next-gen track when we were looking at web components, but I think we're back to "don't use them in new stuff" now. @clinuz, are you on the same page?